src/Compilers/CSharp/Portable/Symbols/TypeSymbol.cs

```diff
@@ -1269,11 +1424,48 @@ private static bool IsExplicitlyImplementedViaAccessors(Symbol interfaceMember,
                     }
                 default:
                     {
-                        implementingMember = null;
-                        return false;
+                        interfaceAccessor1 = null;
+                        interfaceAccessor2 = null;
+                        break;
                     }
             }
 
+            if (!interfaceAccessor1.IsImplementable())
+            {
+                interfaceAccessor1 = null;
+            }
+
+            if (!interfaceAccessor2.IsImplementable())
+            {
+                interfaceAccessor2 = null;
+            }
+
+            return (interfaceAccessor1, interfaceAccessor2);
+        }
+
+        /// <summary>
+        /// Since dev11 didn't expose a symbol API, it had the luxury of being able to accept a base class's claim that 
+        /// it implements an interface.  Roslyn, on the other hand, needs to be able to point to an implementing symbol
+        /// for each interface member.
+        /// 
+        /// DevDiv #718115 was triggered by some unusual metadata in a Microsoft reference assembly (Silverlight System.Windows.dll).
+        /// The issue was that a type explicitly implemented the accessors of an interface event, but did not tie them together with
+        /// an event declaration.  To make matters worse, it declared its own protected event with the same name as the interface
+        /// event (presumably to back the explicit implementation).  As a result, when Roslyn was asked to find the implementing member
+        /// for the interface event, it found the protected event and reported an appropriate diagnostic.  Would it should have done
```

Comment:
>Would [](start = 107, length = 5)

"Would" should be "What"?

Reply:
> "Would" should be "What"?

This is existing comment, it is simply shown as moved by the diff tools.

---
In reply to: [324294700](https://github.com/dotnet/roslyn/pull/38683#discussion_r324294700) [](ancestors = 324294700)

Reply:
Sure, but could you please fix it?

---
In reply to: [324298231](https://github.com/dotnet/roslyn/pull/38683#discussion_r324298231) [](ancestors = 324298231,324294700)

Reply:
> Sure, but could you please fix it?

That I can do. I assume you requesting to replace "Would" with "What".

---
In reply to: [324313423](https://github.com/dotnet/roslyn/pull/38683#discussion_r324313423) [](ancestors = 324313423,324298231,324294700)